src/swap/types.ts

```diff
@@ -298,7 +298,8 @@ export type SwapReact = {
   onError?: (error: SwapError) => void; // An optional callback function that handles errors within the provider.
   onStatus?: (lifecycleStatus: LifecycleStatus) => void; // An optional callback function that exposes the component lifecycle state
   onSuccess?: (transactionReceipt: TransactionReceipt) => void; // An optional callback function that exposes the transaction receipt
-  title?: string; // Title for the Swap component. (default: "Swap")
+  title?: ReactNode; // Title for the Swap component. (default: "Swap")
+  headerLeftContent?: ReactNode; // Header left content for the Swap component (eg. back button).
```

Comment:
Why not just call this the backButton? Or make it into a back button sub component?

Reply:
what other elements are we expecting to be passed through here?

Reply:
that was the original change, but we made it more generic to be able to handle things like logos, tray-open/close-buttons, etc.